lib/visitor.dart

```diff
@@ -84,7 +84,8 @@ class Visitor {
   T getModification<T>(String key, T defaultValue, {bool activate = false}) {
     var ret = defaultValue;
 
-    if (this.modifications.containsKey(key)) {
+    if (!this.decisionManager.isPanic() &&
```

Comment:
Si the code inside the `if` is getting big, I think i'd prefer if we had an early return condition:
```dart
if (this.decisionManager.isPanic() || !this.modifications.containsKey(key)){
   return ret
}
// Do the rest for the OK case
var modification = this.modifications[key];
...

```
What do you think ?